pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
+	Target kapi.ObjectReference
+	// Type is a published service's claim type
+	Type string
+	// Description is a user-defined description of the published service
+	Description string
```

Comment:
For things where description makes sense as a field, we have actual struct fields (c.f. https://github.com/openshift/origin/blob/347cf3326d5c9ac0a29111078b5415064ba5f236/pkg/template/api/v1/types.go#L45)